docs/src/@primer/gatsby-theme-doctocat/nav.yml

```diff
@@ -115,19 +100,26 @@
       url: /components/tooltips
     - title: Truncate
       url: /components/truncate
+- title: Marketing
```

Comment:
@tobiasahlin How do you feel about moving all the marketing styles to their own section:

<img width="495" alt="Screen Shot 2022-01-21 at 20 16 05" src="https://user-images.githubusercontent.com/378023/150518602-f94f6904-f88d-4098-a2b7-63b430ec1c75.png">

👀  [Preview](https://primer-css-52o9pirgi-primer.vercel.app/css/)

Might be nice if they are all together and would fix the currently odd alphabetical order in the nav. E.g. `Marketing typography` coming after `Typography`. Downside: Marketing utilities and components would be combined. Does it matter? And they are also "pushed down" to after the long list of "product" components.

Reply:
@simurai yesss this is great! I think it's OK that components and utilities are combined 🙏